catalog/dags/providers/provider_dag_factory.py

```diff
@@ -323,20 +323,25 @@ def create_report_load_completion(
     ingestion_metrics,
     dated,
 ):
+    is_reingestion_workflow = "reingestion" in dag_id
+
+    op_kwargs = {
+        "dag_id": dag_id,
+        "media_types": media_types,
+        "duration": ingestion_metrics["duration"],
+        "record_counts_by_media_type": ingestion_metrics["record_counts_by_media_type"],
+        "dated": dated,
+        "is_reingestion_workflow": is_reingestion_workflow,
+    }
+
+    if not is_reingestion_workflow:
+        op_kwargs["date_range_start"] = "{{ data_interval_start | ds }}"
+        op_kwargs["date_range_end"] = "{{ data_interval_end | ds }}"
+
```

Comment:
Nit for more idiomatic python:

```suggestion
    is_reingestion_workflow = "reingestion" in dag_id

    op_kwargs = {
        "dag_id": dag_id,
        "media_types": media_types,
        "duration": ingestion_metrics["duration"],
        "record_counts_by_media_type": ingestion_metrics["record_counts_by_media_type"],
        "dated": dated,
        "is_reingestion_workflow": is_reingestion_workflow,
    }

    if not is_reingestion_workflow:
        op_kwargs = op_kwargs | {
            "date_range_start": "{{ data_interval_start | ds }}",
            "date_range_end": "{{ data_interval_end | ds }}",
        }

```